File: app/src/main/java/com/aaron/yespdf/preview/PagerAdapter.java
package com.aaron.yespdf.preview;

import android.annotation.SuppressLint;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentPagerAdapter;

import com.shockwave.pdfium.PdfDocument;

import java.util.List;

/**
 * @author Aaron aaronzzxup@gmail.com
 */
class PagerAdapter extends FragmentPagerAdapter {

    @SuppressLint("WrongConstant")
    PagerAdapter(@NonNull FragmentManager fm) {
        super(fm, FragmentPagerAdapter.BEHAVIOR_SET_USER_VISIBLE_HINT);
    }

    @NonNull
    @Override
    public Fragment getItem(int position) {
        if (position == 0) {
            return ContentFragment.newInstance();
        }
        return BookmarkFragment.newInstance();
    }

    @Override
    public int getCount() {
        return 2;
    }
}

File: app/src/main/java/com/aaron/yespdf/preview/PagerAdapter.kt
package com.aaron.yespdf.preview

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

/**
 * @author Aaron aaronzzxup@gmail.com
 */
internal class PagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm, BEHAVIOR_SET_USER_VISIBLE_HINT) {

    override fun getItem(position: Int): Fragment {
        return if (position == 0) {
            ContentFragment.newInstance()
        } else BookmarkFragment.newInstance()
    }

    override fun getCount(): Int {
        return 2
    }
}